### PR TITLE
Support for fog_path_style config option

### PR DESCRIPTION
**Pull request by josacar**
Support for fog_path_style config option

[Review on Reviewable](https://reviewable.io/reviews/rumblelabs/asset_sync/302)

### CODE REVIEW

File: spec/unit/asset_sync_spec.rb [hunk 1]
```diff
@@ -106,6 +111,10 @@
       expect(AssetSync.config.fog_region).to eq("eu-west-1")
     end
 
+    it "should configure path_style" do
+      AssetSync.config.fog_path_style.should be_truthy
```
Comment (PikachuEXE) [inline spec/unit/asset_sync_spec.rb]: Should use `expect(AssetSync.config.fog_path_style).to be_truthy`

File: spec/unit/asset_sync_spec.rb [hunk 2]
```diff
@@ -50,6 +51,10 @@
...
```
Comment (PikachuEXE) [inline spec/unit/asset_sync_spec.rb]: Should use `expect(AssetSync.config.fog_path_style).to be_truthy`

**Comment by PikachuEXE** [comment]
Please also change README for this new option
Including a link to the doc of `path_style` in `fog`
I can't find it :(

**Comment by josacar** [comment]
Done. Let me know if something else can be improved.

Thanks in advance.

**Comment by PikachuEXE** [comment]
In the changed README
It is mentioned that `fog_path_style` will be included in file generated by the generator
But I am unable to find any change related

To make this PR easier to be merged 
I think removing the change about generator is better

**Comment by josacar** [comment]
@PikachuEXE Ups, updated. Just kept the fog optional options

**Comment by PikachuEXE** [comment]
Thanks ;)

**Comment by PikachuEXE** [comment]
The new option seems to be AWS only right?
I guess I should move the entry about new option under AWS section

**Comment by josacar** [comment]
AFAIK is `fog-aws` specific but not sure if it can be applied to another providers.

Thanks.

**Pull request merged**